content/docs/dvclive/api-reference/live/index.md

```diff
@@ -38,11 +38,11 @@ other metadata.
 - `dir` - Location of the directory to store
   [outputs](/doc/dvclive/get-started#outputs).
 
-- `summary_path` - `{dir}.json`. Location of the
+- `summary_path` - `{Live.dir}.json`. Location of the
   [summary](/doc/dvclive/api-reference/live/log#description).
```

Comment:
Hola. Why this change? Was just `dir` incorrect? It's listed as just `dir` above.

> Seems unrelated to https://github.com/iterative/dvclive/pull/266 too.

Reply:
> Seems unrelated to https://github.com/iterative/dvclive/pull/266 too.

It has nothing to do, just something I noticed during this P.R. Could be extracted to other